geoportal/c2cgeoportal_geoportal/scaffolds/create/alembic.ini.mako_tmpl

```diff
@@ -1,32 +1,21 @@
-# A generic, single database configuration.
+[DEFAULT]
+script_location = /opt/alembic
```

Comment:
This will only works with docker, but c2cgeoportal-commons:alembic would works everywhere

Reply:
Effectively, fixed :-)

Reply:
Finally I realyse that it's already working and tested, executed in docker but working in non docker environment :-)